client_demo/start.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+
+"""A script to start Server && Client throught launch file and to control the restart of Server"""
+from threading import Thread
+import time 
+import os 
+import sys
+import signal 
+import psutil
+
+signal.signal(signal.SIGINT, signal.SIG_DFL)
+sleep_time = 3
+launchCmd = 'ros2 launch client_demo test.launch.py'
+
+def restart_server():
+    try:
+        time.sleep(sleep_time)
+        print("parameter server restart now...")
+        program_name = 'server'
+        for process in psutil.process_iter():
+            if process.name() == program_name:
+                path = psutil.Process(process.pid)
+                if "install/parameter_server/lib/parameter_server" in path.exe():
```

Comment:
this does not work when it uses `colcon build --symlink-install`, i think that most common case is this test will be used on mainline with source build with `colcon build --symlink-install`. what do you think? the following solves this problem.

```suggestion
                if "install/parameter_server/lib/parameter_server" or "parameter_server/server" in path.exe():
```

Reply:
Make sense to me,  already changed.